x/contractmanager/keeper/grpc_query_failure_test.go

```diff
@@ -127,6 +128,10 @@ func TestFailureQueryPaginated(t *testing.T) {
 			nullify.Fill(resp.Failures),
 		)
 	})
+	t.Run("More than limit", func(t *testing.T) {
+		_, err := keeper.Failures(wctx, request(nil, 0, math.MaxUint64, true))
+		require.Error(t, err)
```

Comment:
Better to check that you get a particular error, not some error in general

Reply:
added validation by error text